tests/e2e/login/purge.spec.js

```diff
@@ -236,7 +236,11 @@ describe('Purging on login', () => {
     return browser.executeAsyncScript((() => {
       const callback = arguments[arguments.length - 1];
       const db = window.PouchDB('medic-user-e2e_restricted');
-      db.get('_local/purgelog').then(doc => callback(doc), err => callback(err));
+      db.get('_local/purgelog')
+        .then(doc => callback(doc), err => callback(err))
+        .catch(() => {
+          process.exit(1);
+        });
```

Comment:
I think there's the same historical formatting bug again... the second param to the `.then` isn't working. It's also weird because usually callbacks take the `err` as the first param and the `result` as the second param, but this one is being passed both in the first param? I haven't looked into it but it may need some investigation to work out how we're using it.

This is what I'd expect:

```suggestion
      db.get('_local/purgelog')
        .then(doc => callback(null, doc))
        .catch(err => callback(err));
```